client/a8c-for-agencies/sections/sites/sites-dashboard/provisioning-site-notification.tsx

```diff
@@ -0,0 +1,51 @@
+import NoticeBanner from '@automattic/components/src/notice-banner';
+import { useTranslate } from 'i18n-calypso';
+import { useState } from 'react';
+import useIsSiteReady from 'calypso/a8c-for-agencies/data/sites/use-is-site-ready';
+
+type Props = {
+	siteId: number;
+};
+
+export default function ProvisioningSiteNotification( { siteId }: Props ) {
+	const { isReady, site } = useIsSiteReady( { siteId } );
+	const [ showBanner, setShowBanner ] = useState( true );
+
+	const translate = useTranslate();
+
+	return (
+		showBanner && (
+			<NoticeBanner
+				level={ isReady ? 'success' : 'warning' }
+				hideCloseButton={ ! isReady }
+				onClose={ () => setShowBanner( false ) }
+				title={
+					isReady
+						? translate( 'Congratulation on your new WordPress.com site!' )
```

Comment:
Small typo:
Congratulation → Congratulations

Reply:
Good catch. Thank you for flagging it.

Reply:
I created a PR to fix this: https://github.com/Automattic/wp-calypso/pull/90388